services/appflowy-worker/src/import_worker/worker.rs

```diff
@@ -987,15 +985,6 @@ async fn process_unzip_file(
         err
       ))
     })?;
-    // Update the workspace database cache because newly created workspace databases are cached in Redis.
-    mem_cache
-      .insert_encode_collab(
```

Comment:
@appflowy I think removing this part is the reason why 2 tests are failing but why? After all men_cache is just a cache. If it's missing collab lookup should fallback to postgres/s3, right?

Reply:
it's used to update existing object that was cached in redis. `insert_into_af_collab_bulk_for_user` doesn't update corresponding redis cache. `insert_into_af_collab_bulk_for_user` should not update the cache because not all objects will be used immediatly.